tests/unit/readers/cubeconfig/v1_0/test_build_friendly_error_msgs.py

```diff
@@ -494,33 +456,6 @@ def test_val_errors_undefined_unit_uri():
     )
 
 
-def test_val_errors_uri_conflict():
```

Comment:
Why has this one been deleted?

Reply:
This was deleted by mistake and added back again now.